File: GTTS/INL.TrainingService.Models/InterAgencyAgreementsAtBusinessUnit_Item.cs

namespace INL.TrainingService.Models
{
    public class InterAgencyAgreementsAtBusinessUnit_Item : IInterAgencyAgreementsAtBusinessUnit_Item
    {
        public int InterAgencyAgreementID { get; set; }
        public string Code { get; set; }
        public string Description { get; set; }
        public long BusinessUnitID { get; set; }
        public string Acronym { get; set; }
        public string BusinessUnitName { get; set; }
        public bool BusinessUnitActive { get; set; }
        public bool InterAgencyAgreementBusinessUnitActive { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/inter-agency-agreements-at-business-unit_item.ts


import { IInterAgencyAgreementsAtBusinessUnit_Item } from './iinter-agency-agreements-at-business-unit_item';

export class InterAgencyAgreementsAtBusinessUnit_Item implements IInterAgencyAgreementsAtBusinessUnit_Item {
  
	public InterAgencyAgreementID: number = 0;
	public Code: string = "";
	public Description: string = "";
	public BusinessUnitID: number = 0;
	public Acronym: string = "";
	public BusinessUnitName: string = "";
	public BusinessUnitActive: boolean = false;
	public InterAgencyAgreementBusinessUnitActive: boolean = false;
  
}






